src/content/AnimationLibrary.cpp

```diff
@@ -94,8 +94,7 @@ namespace Animations
 
     Handle::AnimationDataHandle AnimationLibrary::loadMAN(const std::string& name)
     {
-        std::string file_name = name + ".MAN";
-        std::transform(file_name.begin(), file_name.end(), file_name.begin(), ::toupper);
+        std::string file_name = Utils::uppered(name + ".MAN");
```

Comment:
I think engine should have tool for dealing with case sensitivity, instead of hardcodes.

Reply:
The original engine is case insensitive as it converted all of the file names and indentifiers to uppercase internally, I believe this just mirrors it.